css-box-model/styles.css

```diff
@@ -0,0 +1,12 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+}
+
+button {
+  margin: 0 12px;
+  padding: 6px 16px;
+  border-radius: 6px;
+}
```

Comment:
Instead of making your own rule, you should add the properties to the above rule under the "Your code here" comment.